src/main/java/io/github/sefiraat/networks/slimefun/network/NetworkQuantumStorage.java

```diff
@@ -108,10 +119,12 @@ public class NetworkQuantumStorage extends SlimefunItem implements DistinctiveIt
 
     private final List<Integer> slotsToDrop = new ArrayList<>();
     private final int maxAmount;
+    private final boolean supportsCustomMaxAmount;
```

Comment:
(Just personal preference so interested in what y'all think) You could rename this to customMax  just because super long names like that are... icky

Reply:
imo customMax makes it seem like that should be the int value that actually contains the custom max, instead of the flag of it being supported or not